subsys/bluetooth/rpc/common/serialize.h

```diff
@@ -285,13 +285,22 @@ void *ser_decode_callback_call(CborValue *value);
 
 /** @brief Put decoder into an invalid state and set error code that caused it.
  *         All further decoding on this decoder will be ignored.
- *         Invalid state can be checked with the is_decoder_invalid() function.
+ *         Invalid state can be checked with the ser_decode_valid() function.
```

Comment:
```suggestion
 *         Invalid state can be checked with the @ref ser_decode_valid() function.
```

Reply:
Links to functions still not working, I believe (https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_styleguide.html#references).